### PR TITLE
Bump to the latest Guava version to avoid CVEs and be up to date

### PR DESCRIPTION
**Pull request by casewalker**
Upgrades past CVE-2023-2976 and CVE-2020-8908 to the latest [Guava version](https://mvnrepository.com/artifact/com.google.guava/guava).

### CODE REVIEW

**Comment by casewalker** [comment]
@mikesamuel If you could take a look, that would be greatly appreciated.

**Comment by melloware** [comment]
+1

**Comment by maxibarros** [comment]
+1 @mikesamuel

File: parent/pom.xml [hunk 1]
```diff
@@ -88,7 +88,7 @@ application while protecting against XSS.
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <guava.version>30.1-jre</guava.version>
+    <guava.version>32.1.3-jre</guava.version>
```
Comment (melloware) [inline parent/pom.xml:91]: 33.0.0-jre

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by subbudvk** [comment]
I think dependabot can create such PR. Saw one for another version - https://github.com/OWASP/java-html-sanitizer/pull/284. Also there is some discussion on removing the dependency https://github.com/OWASP/java-html-sanitizer/pull/272

**Comment by casewalker** [comment]
@subbudvk If you read the comments on the PR you linked and check the Guava link I shared above, you'll see that the Dependabot PR is trying to upgrade from one vulnerable version to another vulnerable version.

**Comment by casewalker** [comment]
@jmanico Hello, I saw that you recently reviewed a PR in this repo. I am trying to have some CVEs addressed by bumping to the latest version of Guava. If you could review this PR, that would be greatly appreciated.

Thanks!

**Comment by melloware** [comment]
@casewalker this can be closed now that https://github.com/OWASP/java-html-sanitizer/pull/272 has been merged

**Comment by mikesamuel** [comment]
This is obviated by https://github.com/OWASP/java-html-sanitizer/commit/3b6cc1b7e6d7992c5b15f87f91dc163de7d35c05 which removes the guava dependency entirely

**Comment by casewalker** [comment]
Beautiful, thanks for addressing the underlying issue!!

**Pull request closed without merging**